check-deps/Makefile

```diff
@@ -37,7 +37,7 @@ endif
 CFLAGS_cc_has_mfentry = -mfentry
 LDFLAGS_cxa_demangle = -lstdc++
 LDFLAGS_have_libelf = -lelf
-CFLAGS_cc_has_mno_sse2 = -mno-sse -mno-sse2
+CFLAGS_cc_has_mno_sse2 = -O2 -mno-sse -mno-sse2
```

Comment:
Thanks for the fix.  Do you know why it makes the difference?

Reply:
I believe this is a bug of GCC (4.8.5) which SSE support has been disabled with '-mno-sse -mno-sse2' but SSE register still has been used in optimized code (-O2) generation.

See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=99652